pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -143,11 +138,13 @@ func (c *controller) Run(stop <-chan struct{}) {
 }
 
 func (c *controller) ConfigDescriptor() model.ConfigDescriptor {
-	return model.ConfigDescriptor{model.IngressRule}
+	//TODO: are these two config descriptors right?
+	return model.ConfigDescriptor{model.Gateway, model.VirtualService}
 }
 
+//TODO: we don't return out of this function now
```

Comment:
this code have been incorrect.

either we should have a "not implemented error" or we should try to grab a k8s ingress resource using the name and convert it using `ConvertIngressV1alpha3`. @ZackButcher @GregHanson, thoughts?

Reply:
I'd be more comfortable with not implemented but I await other thoughts.